common/static/common/js/utils/require-queue.js

```diff
@@ -0,0 +1,21 @@
+/*
+Helper function used to require files serially instead of concurrently.
+ */
+(function (window, require) {
+  'use strict';
+
+  window.requireQueue = function(modules, callback) {
+    function load(queue, results) {
+      if (queue.length) {
+        require([queue.shift()], function(result) {
+          results.push(result);
+          load(queue, results);
+        });
+      } else {
+        callback.apply(null, results);
```

Comment:
Yes, this particular callback is not expecting the results, but this is a generic helper method.